userdata/cloud-config-controller

```diff
@@ -20,6 +20,8 @@ exec /usr/bin/coreos-cloudinit --from-file /var/run/coreos/$USERDATA_FILE
 coreos:
   update:
     reboot-strategy: "off"
+    group: {{.ClusterName}}-controller
```

Comment:
@mclarke47 @sbuliarca:

Ok, so this should expand out to:

```
pre-prod-eu-pac-controller
prod-eu-upp-controller
```
etc, etc.

Thoughts? Does that work for everyone?

Reply:
(for Matt's reference - UPP and PAC are being considered to be two separate platforms, hence the extra confusion)

Reply:
Indeed I think this is the best option so far. We'll need to remember to include these lines in future kube-aws upgrades, but this should be easily spotted on PRs